### PR TITLE
add index directly and remove nullable false

### LINKED ISSUE #291: Refactor migrations_core_v1 file
- [ ] remove nullable(false) from migrations (fields are not nullable by default) https://github.com/CodingCab/ShipTownDev/blob/af924b455358056d067b4acb50772ac3fc758f8a/database/migrations/2019_10_16_000000_core_v1.php#L256
- [ ] add index option directly to column in creation
sample: $table->string('status_code')->default('')->index();
https://github.com/CodingCab/ShipTownDev/blob/af924b455358056d067b4acb50772ac3fc758f8a/database/migrations/2019_10_16_000000_core_v1.php#L384

### PR DESCRIPTION
**Pull request by robgoodliffe**
closes #291

### CODE REVIEW

File: database/migrations/2019_10_16_000000_core_v1.php [hunk 1]
```diff
@@ -114,11 +114,9 @@ public function up()
             $table->longText('data')->nullable();
         });
 
-
-
         Schema::create('mail_templates', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('code')->nullable(false)->default('');
+            $table->string('code')->nullable()->default('');
```
Comment (ArturHanusek) [inline database/migrations/2019_10_16_000000_core_v1.php:119]: @robgoodliffe this is wrong, it now allows code field to be null... it shouldn't be the case

File: database/migrations/2019_10_16_000000_core_v1.php [hunk 2]
```diff
@@ -236,7 +229,7 @@ public function up()
 
         Schema::create('warehouses', function (Blueprint $table) {
             $table->id();
-            $table->string('code', 5)->nullable(false)->unique();
+            $table->string('code', 5)->nullable()->unique();
```
Comment (ArturHanusek) [inline database/migrations/2019_10_16_000000_core_v1.php:232]: @robgoodliffe same as previous comment

**Review by ArturHanusek — CHANGES_REQUESTED** [review]
->nullable(false) is complete opposite to ->nullable()

            $table->string('code')->nullable(false)->default('');
===
            $table->string('code')->default('');

**Comment by robgoodliffe** [comment]
@ArturHanusek should be fixed now. Apologies. Misunderstanding on my part.

File: database/migrations/2019_10_16_000000_core_v1.php [hunk 3]
```diff
@@ -130,7 +128,7 @@ public function up()
 
         Schema::connection(config('activitylog.database_connection'))->create(config('activitylog.table_name'), function (Blueprint $table) {
             $table->id();
-            $table->string('log_name')->nullable();
+            $table->string('log_name')->index();
```
Comment (ArturHanusek) [inline database/migrations/2019_10_16_000000_core_v1.php:131]: @robgoodliffe last one, log_name should be nullable in that case i believe

**Review by ArturHanusek — CHANGES_REQUESTED** [review]

**Comment by robgoodliffe** [comment]
@ArturHanusek fixed

**Review by ArturHanusek — APPROVED** [review]

**Pull request merged**